phpcs.xml.dist

```diff
@@ -49,7 +49,7 @@
 	<!-- Limit to PHP files -->
 	<arg name="extensions" value="php"/>
 
-	<config name="ignore_warnings_on_exit">true</config>
+	<config name="ignore_warnings_on_exit" value="true"/>
```

Comment:
Strictly speaking, this is not related to this PR, I only added it to trigger the CI build. I can create a new PR with this fix alone.

Reply:
If this is not related to the PR, I'd seaparate it on its own to better track it down the line.

Reply:
Moved this into a separate PR: #2373

Will drop this commit after rebase.

Reply:
Done.